// 11_exception_handling_2.cpp